src/pages/Office/MoveTaskOrder/MoveTaskOrder.jsx

```diff
@@ -716,6 +754,11 @@ export const MoveTaskOrder = (props) => {
               Your changes were saved
             </Alert>
           )}
+          {renderSITAddressUpdateAlert && (
+            <Alert type="warning" headingLevel="h4" className={styles.alertWithButton}>
```

Comment:
```suggestion
            <Alert type="warning"  className={styles.alertWithButton}>
```
removing the heading level should fix the sizing.